qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/QuteSupportForTemplate.java

```diff
@@ -436,6 +437,7 @@ public ResolvedJavaTypeInfo getResolvedJavaType(QuteResolvedJavaTypeParams param
 		resolvedType.setMethods(methodsInfo);
 		resolvedType.setInvalidMethods(invalidMethods);
 		resolvedType.setExtendedTypes(extendedTypes);
+		resolvedType.setKind(JDTTypeUtils.getJavaTypeKind(type));

```

Comment:
As you fill now correctly the kind, could you add this assert for each test like https://github.com/redhat-developer/quarkus-ls/blob/9b5fe029af4c6d2ec2b80240be6d91fbdd3043de/qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGetResolvedJavaTypeTest.java#L52

`Assert.assertEquals(JavaTypeKind.Interface, result.getJavaTypeKind());`